src/imex.rs

```diff
@@ -753,11 +759,14 @@ async fn export_backup_old(context: &Context, dir: impl AsRef<Path>) -> Result<(
         );
     }
     let dest_sql = Sql::new();
-    ensure!(
-        dest_sql.open(context, &dest_path_filename, false).await,
-        "could not open exported database {}",
-        dest_path_string
-    );
+    dest_sql
+        .open(context, &dest_path_filename, false)
+        .await
+        .context(format!(
```

Comment:
with_context takes a closure and is preferrable here

Reply:
of course you're right